src/main/kotlin/ink/abb/pogo/scraper/Bot.kt

```diff
@@ -112,8 +113,11 @@ class Bot(val api: PokemonGo, val settings: Settings) {
                 task(catch)
             if (settings.shouldDropItems)
                 task(drop)
-            if (settings.shouldAutoTransfer)
+            if (settings.shouldAutoTransfer) {
                 task(release)
+            } else {
+                task(smartEvolve)
```

Comment:
It happened to fit what I was planning on at the time. I am considering the final touches to make this all configurable now.